Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -2546,6 +2547,44 @@ bool CSSPropertyParser::consumeListStyleShorthand(bool important)
     return m_range.atEnd();
 }
 
+bool CSSPropertyParser::consumeWhiteSpaceShorthand(bool important)
+{
+    auto ident = consumeIdentRaw(m_range);
+    if (!ident)
+        return false;
+    if (!m_range.atEnd())
+        return false;
+
+    // FIXME: accept values other than the shorthand keywords
+    // FIXME: set whiteSpaceTrim values
```

Comment:
```suggestion
    // FIXME: Implement `<'white-space-collapse'> || <'text-wrap'>` syntax.
```

I'd remove the whiteSpaceTrim fixme for now, since I'd not specific to this function.